data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -156,3 +156,9 @@ while true; do
     done
     sleep 10
 done
+
+until oc get nodes; do
```

Comment:
The loop above is infinite, you'll never reach this code

Reply:
Yeah sorry.. didn't mean for that to be in the commit.